backend/stewards/views.py

```
@@ -1,23 +1,164 @@
+        and (user.is_staff or user.is_superuser)
+    )
+
+
+class FeatureCandidateReviewViewSet(viewsets.ViewSet):
+    """
+    Blind scoring view for interesting submissions.
+    Reviewers see only their own scores. Staff can access aggregate decisions.
+    """
+    permission_classes = [IsAuthenticated]
+
+    def _interesting_queryset(self):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_interesting_queryset`

(ANN202)